gadget/mkfs.go

```diff
@@ -33,18 +33,26 @@ import (
 // filesystem label, and populates it with the contents of provided root
 // directory.
 func MkfsExt4(img, label, contentsRootDir string) error {
```

Comment:
Maybe a comment here as well that this will create filesystems that will not work on UC16.